orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/FindImageFromClusterTask.groovy

```diff
@@ -205,14 +207,19 @@ class FindImageFromClusterTask extends AbstractCloudProviderAwareTask implements
           // the image into the target account
           List<Map> defaultImages = oortService.findImage(cloudProvider, searchNames[0] + '*', defaultBakeAccount, null, null)
           resolveFromBaseImageName(defaultImages, missingLocations, imageSummaries, deploymentDetailTemplate, config, stage.execution.id)
-          def stillUnresolved = imageSummaries.findResults { it.value == null ? it.key : null }
-          if (stillUnresolved) {
-            throw new IllegalStateException("Missing images in $stillUnresolved.value")
-          }
-        } else {
-          throw new IllegalStateException("Missing images in $unresolved.value")
+          unresolved = imageSummaries.findResults { it.value == null ? it.key : null }
         }
       }
+
+      if (unresolved) {
+        def errorMessage = "Missing image '${searchNames[0]}' in regions: $unresolved.value"
```

Comment:
We don't need to wrap `${unresolved.value}`?

Reply:
good catch, thanks!